src/pwr_mgmt/pwr.go

```diff
@@ -29,24 +29,23 @@ func (w ReaderWriter) WriteScalingGov(sclgov string, cpu int) error {
 	}
 	scalingGovFile := fmt.Sprintf(w.ScalingGovernorPath, cpu)
 
-	err := os.WriteFile(scalingGovFile, []byte(sclgov), 0644)
+	err := os.WriteFile(scalingGovFile, []byte(sclgov), 0)
```

Comment:
Instead of cheating the static code analyzer, a better way to do this is to open the existing file for write only, then write to it.
```
f, err := os.OpenFile(scalingGovFile, os.O_WRONLY, 0)
if err != nil {
    return fmt.Errorf("error opening %s: %v", scalingGovFile, err)
}
defer f.Close()

_, err = f.Write([]byte(sclgov))
if err != nil {
    return fmt.Errorf("error writing to %s: %v", scalingGovFile, err)
}
```